sklearn/utils/sparsefuncs_fast.pyx

```diff
@@ -131,13 +131,19 @@ def _csr_mean_variance_axis0(np.ndarray[floating, ndim=1, mode="c"] X_data,
         np.ndarray[floating, ndim=1] sum_weights_nz = \
             np.zeros(shape=n_features, dtype=dtype)
 
+        np.ndarray[np.uint64_t, ndim=1] counts = np.full(
+            fill_value=weights.shape[0], shape=n_features, dtype=np.uint64)
```

Comment:
```suggestion
            fill_value=n_samples, shape=n_features, dtype=np.uint64)
```

Reply:
they might actually be different. n_samples is not used in this function, I've planed to make a subsequent PR to clean this unused param.